dokka-subprojects/plugin-base/src/test/kotlin/markdown/LinkTest.kt

```diff
@@ -1118,6 +1119,37 @@ class LinkTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun `should have a correct KDoc and external links when there is a newline inside link text`() {
+        testInline(
+            """
+            |/src/main/kotlin/Testing.kt
+            |/**
+            |* Text
+            |* [some 
+            |* link](https://www.google.com/)
+            |*
+            |* Text:
+            |* [some
+            |*  declaration][usage]
+            |*/
+            |fun usage() = 0
+            |}
+        """.trimMargin(),
+            configuration
+        ) {
+            documentablesMergingStage = { module ->
+                val doc = module.dfs { it.name == "usage" }?.documentation?.values?.single()
+                    ?: throw IllegalStateException("Can't find documentation for declaration 'usage'")
+                val externalLink = doc.firstMemberOfType<A>()
+                assertEquals("some link", externalLink.children.first().text())
```

Comment:
it would be nice to also check, that we parsed the full link correctly, including url (`https://www.google.com/`)
Same for the `docLink` below